lib/MusicBrainz/Server/Constants.pm

```diff
@@ -1073,6 +1077,29 @@ Row ID for the Deleted Artist entity
 
 Row ID and GID for the special label "[no label]"
 
+=item $MAX_ONELINE_STRING_LENGTH
+
+Maximum number of characters for a one-line string that is otherwise unlimited
+in the database schema.
+
+It is set to 1024, initially in order to prevent MBS-13536 from happening
+again, because multiple-bytes characters in Postgres can take up to 4 bytes
+and the maximum size for the index type is affected by the block size which
+is generally set to 8192 bytes when compiling Postgres, and also because no
+legitimate examples have been found or are to be expected beyond this length.
+
+The main reason is to bring a reasonable bound for otherwise unlimited titles,
+not only to prevent index issues, but also to make it easier to use for other
+pieces of software accessing this data, and even to improve the data quality.
+
+=item $MAX_POSTGRES_INDEXED_STRING_BYTES
+
+Maximum number of bytes for a string to be indexable by Postgres.
```

Comment:
In case you feel that it's worth mentioning, it's technically specific to btree indexes, but they are the default type of index and we don't use the other types extensively.

Reply:
Totally. Now mentioned. See https://github.com/metabrainz/musicbrainz-server/compare/dfd7f9bca469279bb32ad50d301811a5035b92bd..b0a32248ef6a7e3ccafc18ce771d31bba3e4c5a7